doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Sonata project contribution
+
+Thanks for you interest onto Sonata projects!
+
+## Summary
+
+* [Issues]()
+* [Pull Requests](#pull-requests)
+* [Label rules]()
+
+## Pull Requests
+
+All the sonata team will be glad to review your code changes propositions! :smile:
+
+But please, read the following before.
+
+### Coding style
+
+Each project follows [PSR-1](http://www.php-fig.org/psr/psr-1/), [PSR-2](http://www.php-fig.org/psr/psr-2/)
+and [Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html) for coding style,
+[PSR-4](http://www.php-fig.org/psr/psr-4/) for autoloading.
+
+Please [install PHP Coding Standard Fixer](http://cs.sensiolabs.org/#installation)
+and run this command before committing your modifications:
+
+```bash
+php-cs-fixer fix --verbose
+```
+
+### Writing a Pull Requests
```

Comment:
I would add that when you contribute a PR, it's often to fix your breaking application. You want it to work.
When you change sthg, sometimes it triggers an edge-case in another part of the code, that becomes a blocker that needs fixin'.
You can't wait weeks for this small fix to be merged before going on and fixing the real issue.
You can't have two parrallel branches because possible conflicts _and you need your app to work while waiting for the merge_.

There's also the opposite case, when after you changed sthg, your code is so utterly perfect that a feature you thought impossible becomes a one-liner, so you add it in the same PR... :bowtie: